bedrock/settings/static_media.py

```diff
@@ -1017,6 +1019,8 @@
             'js/base/mozilla-client.js',
             'js/base/mozilla-image-helper.js',
             'js/base/nav-main-resp.js',
+            'js/base/class-list-polyfill.js',
```

Comment:
I'm adding this to support IE 9 & 10, although I could be tempted to drop the polyfill if we don't think it's worth it.